app/src/App/App.tsx

```diff
@@ -18,10 +19,12 @@ function App() {
             <Route
               path={"/" + classPrefix + classNumber}
               element={<ClassList classLevelNumber={classNumber[0]} />}
-            />
+            >
+            </Route>
```

Comment:
Why change this route from self-closing to not, when the others aren't?

Reply:
Oh woops I was playing around with stuff and forgot to change that back. Fixed!